cli/src/commands/utils.ts

```diff
@@ -85,3 +114,40 @@ export function validateAddress(address: string) {
     process.exit(1)
   }
 }
+
+interface WalletConfig {
+  wallet: string
+  password: string
+}
+
+interface Config {
+  wallet_config: WalletConfig
+  node_url: string
+  chunk_size: number
+  secret_key: string
+}
+
+export function parseConfigFile(filePath: string): Config {
+  const fileContent = readFileSync(filePath, 'utf-8')
+  try {
+    const parsedData = JSON.parse(fileContent)
+    return parsedData
```

Comment:
I guess we could simplify like that as we don't use the variable except for returning it ?
```suggestion
    return JSON.parse(fileContent)
```